lib/node/client.ml

```diff
@@ -26,8 +24,9 @@ let create_request node recipient payload =
             category = Message.Request;
             sub_category_opt = None;
             id = !id;
+            timestamp = Unix.gettimeofday ();
             sender = !node.address;
-            recipient;
+            recipients = [recipient];
```

Comment:
From my point of view, this is not a good change.
In Deku side _(@aguillon and @AngryStitch please correct me if I am wrong)_ we want to either 
- send a Request (to a specific node, and we care about the result)
- broadcast an information (to the whole network, and we don't care about the result)
This is what I understand from `network.ml` file on main branch.

Hence, in Pollinate, we should provide these two features, which are totally different.
1. Sending request/response
2. Broadcasting message (Post in this PR).

All that to say, since the recipient is contained in the message, `create_request` and `create_post` should not return the same kind of Message.
I think I would have use GADT in Haskell for this, but don't know about OCaml :/

Reply:
Let's open a dedicated issue for that. This is absolutely not a blocker.